.github/workflows/ci_cd.yml

```
@@ -822,13 +822,33 @@ jobs:
       - name: Run Playwright tests
         run: just frontend/run ${{ matrix.script }} --workers=2
 
+      - name: Check for changes after running tests with -u
+        id: check_snapshots_changed
+        if: success() || failure()
+        run: |
+          if [ -n "$(git diff --binary --name-only)" ]; then
+            git add . && git diff --staged --binary | gzip -f > /tmp/snapshot_diff.patch.gz
```

Let's skip this compression.
```suggestion
            git add . && git diff --staged --binary > /tmp/snapshot_diff.patch.gz
```